magenta/interfaces/midi/magenta_midi.py

```diff
@@ -89,7 +89,9 @@
 tf.app.flags.DEFINE_integer(
     'temperature_control_number',
     None,
-    'The control change number to use for controlling softmax temperature.')
+    'The control change number to use for controlling softmax temperature.'
+    'This should be an integer between 0 and 127 corresponding to a softmax '
+    'temperature between 0.1 and 2.')
```

Comment:
Love it. That makes much more sense and should help temperature fiddlers, like myself, to work with this interface. I will propose that change ASAP.